fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/RelOpValue.java

```diff
@@ -89,11 +87,11 @@ public abstract class RelOpValue extends AbstractValue implements BooleanValue {
     private final Iterable<? extends Value> children;
 
     @Nonnull
-    private static final Supplier<Map<Pair<Comparisons.Type, Type.TypeCode>, UnaryPhysicalOperator>> unaryOperatorMapSupplier =
+    private static final Supplier<Map<UnaryOperatorSignature, UnaryPhysicalOperator>> unaryOperatorMapSupplier =
```

Comment:
Only `private` access, but I went with a real class here mainly to make the `Triple` replacement work in an analogous way